doc/RPC.md

```diff
@@ -0,0 +1,1356 @@
+          "avg_tp_wait_duration": 0,
+          "total_count": 12227,
+          "total_processing_duration": 13878697137699,
+          "total_size": 7258242,
+          "total_tp_wait_duration": 0,
+          "type": "DagBlockPacket"
+        },
+        {
+          "avg_processing_duration": 37547,
+          "avg_size": 616,
+          "avg_tp_wait_duration": 0,
+          "total_count": 28,
+          "total_processing_duration": 1051340,
+          "total_size": 17257,
+          "total_tp_wait_duration": 0,
+          "type": "GetDagSyncPacket"
+        },
+        {
+          "avg_processing_duration": 1283007536,
+          "avg_size": 183,
+          "avg_tp_wait_duration": 0,
+          "total_count": 6593,
+          "total_processing_duration": 8458868688722,
+          "total_size": 1206519,
+          "total_tp_wait_duration": 0,
+          "type": "PbftBlockPacket"
+        },
+        {
+          "avg_processing_duration": 1396962902,
+          "avg_size": 231,
+          "avg_tp_wait_duration": 0,
+          "total_count": 236402,
+          "total_processing_duration": 330244824029935,
+          "total_size": 54608862,
+          "total_tp_wait_duration": 0,
+          "type": "VotePacket"
+        },
+        {
+          "avg_processing_duration": 1265584267,
+          "avg_size": 165,
+          "avg_tp_wait_duration": 0,
+          "total_count": 41396,
+          "total_processing_duration": 52390126337007,
+          "total_size": 6863629,
+          "total_tp_wait_duration": 0,
+          "type": "TransactionPacket"
+        },
+        {
+          "avg_processing_duration": 30338,
+          "avg_size": 8,
+          "avg_tp_wait_duration": 0,
+          "total_count": 2,
+          "total_processing_duration": 60676,
+          "total_size": 16,
+          "total_tp_wait_duration": 0,
+          "type": "GetVotesSyncPacket"
+        },
+        {
+          "avg_processing_duration": 1381111831,
+          "avg_size": 17,
+          "avg_tp_wait_duration": 0,
+          "total_count": 508,
+          "total_processing_duration": 701604810378,
+          "total_size": 8636,
+          "total_tp_wait_duration": 0,
+          "type": "StatusPacket"
+        }
+      ]
+    }
+  }
+}
+```
+
+### get_all_nodes
+
+Returns info of current connected nodes including its id, ip_address and port 
+
+#### Parameters
+
+none
+
+#### Returns
+
+`STRING` - string that contains info about peer if format `id ip_address:port` separated with `\n`.
+
+#### Example 
+
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"get_all_nodes","params":[],"id":1}'
+
+// Result
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": "f8ff62f2df819566023ec3cda97f73697e789089b5c97c22a24c175edec3256242796a7a843fe1b949558160dae0fd5510e31c93440798212ef99a55c64e7b0f 66.94.114.186:10002\n45022ea1765d3a569003e2088a0893ffe00656c25f32ee414b39c6e45c8aa05cc41453158d832c67b5bba710edb648223f654fae9e2a1f513d7e42447f25a2d6 38.242.218.203:10002"
```

Comment:
hmm our delimiter in this response is \n ? :D

Reply:
Yeah. I think we need to change it, but in separate micro-task maybe. Because I wanted to discuss this on meet